src/components/Comments.jsx

```diff
@@ -1,12 +1,22 @@
 import { useState, useEffect } from "react";
+import { getComments } from '../api';
 
 
-export const Comments = () => {
+export const Comments = ({ review_id }) => {
     const [comments, setComments] = useState([]);
 
```

Comment:
I'd still consider having an is loading design pattern for the comments while they're being fetched